INSTALL.md

```diff
@@ -0,0 +1,117 @@
+## Pre Requirements
+
+- Access to a Azure subscription of the customer with `Owner` permissions (service principal or human user)
+- Successfully authenticated with `az` CLI
+- GitHub team [service-accounts](https://github.com/orgs/cloudeteer/teams/service-accounts) must be `admin` of the squad repo
+- [Firewall approvals](./docs/firewall-approvals.md)
+
+## Install CDT IaC Launchpad
+
+> [!NOTE]
+> Do the following steps first, then you can use this module with only a few variables as we show in our [example](https://github.com/cloudeteer/terraform-azurerm-launchpad/blob/main/examples/usage/main.tf).
+
+### 1. Create GitHub Personal Access Token (PAT)
+
+The Launchpad Virtual Machine needs permission to register GitHub runners in the squad repository. To be able to do this automatically, a GitHub Personal Access Token (PAT) is required.
+
+Login as GitHub user [cloudeteerbot](https://keepersecurity.eu/vault/#detail/4y33tgsK_Yuv8mJUXDUPrQ) and create a organization-owned GitHub PAT with the following configuration:
```

Comment:
generalize Guide. It should not be cloudeteer specific

Reply:
When creating a generalized INSTALL.md document, it’s a good practice to follow the documented steps and perform the installation yourself to ensure everything works as expected.